src/csharp/Intel/Generator/Assembler/CSharp/CSharpAssemblerSyntaxGenerator.cs

```diff
@@ -173,6 +175,33 @@ public CSharpAssemblerSyntaxGenerator(GeneratorContext generatorContext)
 			}
 		}
 
+		void GenerateDeclareByteArray(FileWriter writer) {
```

Comment:
Why generating code while you can directly add the method to [`Assembler.cs`](https://github.com/0xd4d/iced/blob/master/src/csharp/Intel/Iced/Intel/Assembler/Assembler.cs)? The class is partial for that reason

Reply:
It might be more readable when all `dX` overloads are in the same place. Also perhaps someone might want to improve this in the future to also support other types than `byte`.

Reply:
It is lot less readable when you have a code generator for something that doesn't require code generator elements. The db methods are in the code generator because there are lots of methods and it's easier to generate them there, otherwise, we have already several methods in Assembler that are using existing methods in `Assembler.g.cs`. There is no reason to put it in the generator, it is more cumbersome to maintain.
Partial class are precisely to allow to not put everything in the same place.

It would prefer that you move the code generated in `Assembler.cs` and remove the generator part.

Reply:
Waiting for 0xd4d opinion about it